nautobot/core/constants.py

```diff
@@ -32,6 +32,24 @@
 
 FILTER_NEGATION_LOOKUP_MAP = {"n": "exact"}
 
+FILTER_LOCAL_FIELDS = {
+    "BigIntegerField",
+    "CharField",
+    "DateField",
+    "DateTimeField",
+    "DecimalField",
+    "FloatField",
+    "IntegerField",
+    "PositiveIntegerField",
+    "PositiveSmallIntegerField",
+    "SlugField",
+    "SmallIntegerField",
+    "TextField",
+    "UUIDField",
+}
```

Comment:
Instead of explicitly looking for exact matches on `field.get_internal_type()` strings, can we instead use `isinstance()` checks? That would automatically (and more correctly) handle subclasses of any of these types, and would also simplify this listing substantially (e.g. `isinstance(IntegerField)` would also automatically cover `SmallIntegerField`, `PositiveSmallIntegerField`, etc. since they're just subclasses of it).